src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_config.json

```diff
@@ -1,11 +1,4 @@
 [
-  {
-    "op": "add",
-    "path": "/ValueTypes/AWS::Config::ConfigurationRecorder.ResourceTypes",
```

Comment:
think the references in https://github.com/aws-cloudformation/cfn-lint/blob/main/src/cfnlint/data/ExtendedSpecs/all/04_property_values/aws_config.json would also need to be removed

(also wondering if we should have a test to catch that)

Reply:
the only test we have for this is on the spec itself.  I updated the spec in this PR now too based on the changes so we should have this tested now.